### PR TITLE
Recharge upgrades can only be used once every 7 minutes as borg.

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
When you use a recharge pack as a cyborg, you get the quick charged status effect for seven minutes. While you have this status effect you cannot activate another recharge pack. This also refactors some code to make this work

## Why's this needed? 
For powerborging, the downside of recharging can practically be avoided through a teleporter upgrade and a charging dock loaded with recharge packs. This makes charging packs a compliment to charging instead of a replacement

## Changelog 

```changelog
(u)Ikea
(*)There is now a seven minute cooldown for using cyborg recharge upgrades 
```

### CODE REVIEW

File: code/modules/status_system/statusEffects.dm [hunk 1]
```diff
@@ -2335,6 +2335,15 @@
 #undef LAUNDERED_COLDPROT_AMOUNT
 #undef LAUNDERED_STAIN_TEXT
 
+/datum/statusEffect/quickcharged
+	id = "quick_charged"
+	name = "Quick charged"
+	maxDuration = 7 MINUTES
+
```
Comment (Tarmunora) [inline code/modules/status_system/statusEffects.dm:2342]: does this have an icon?

Comment (ThatFiveGuys) [reply code/modules/status_system/statusEffects.dm:2342]: Oh shoot forgot to add that uhh one second

**Comment by BatElite** [comment]
What problem needs solving here? Getting a teleport + recharges sounds like an active effort on the borg's part to gain benefit, so good on them?

**Comment by Goat-Real** [comment]
Are we trying to solve an issue of cyborgs using the recovery upgrade a lot? Maybe a different idea would be adding a delay on the teleporter upgrade. Change it from an instant vanish button to an escape with couple seconds of charge time.

**Comment by ThatFiveGuys** [comment]
> Are we trying to solve an issue of cyborgs using the recovery upgrade a lot? Maybe a different idea would be adding a delay on the teleporter upgrade. Change it from an instant vanish button to an escape with couple seconds of charge time.

This would be a good change but outside the scope of this pr

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Pull request merged**